src/core/grade_notifier.py

```diff
@@ -25,7 +25,7 @@
 from login_flow.loginState import LoginState
 from helper.message import Message
 from helper.gpa import GPA
-from helper.constants import instance_path, abs_repo_path
+from helper.constants import instance_path, abs_repo_path, SESSION_ENDED_TEXT
```

Comment:
Why is this the only constant imported here? Just do helper.constsnts.SESSION_ENDED_TEXT or is that not possible?

Reply:
then we need to also include the line
`import helper`
in order to later use
`helper.constants.SESSION_ENDED_TEXT`

Reply:
Oh okay fine fine, makes sense